engine/profiler/src/profiler_linux.cpp

```diff
@@ -12,118 +12,25 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-#include <dlib/time.h>
-#include <unistd.h>
-
 #include "profiler_private.h"
-
-// Inspired by this post: http://stackoverflow.com/questions/1420426/how-to-calculate-the-cpu-usage-of-a-process-by-pid-in-linux-from-c
-static uint64_t _sample_cpu_last_t = 0;
-static long int _sample_cpu_last_tot = 0;
-static long int _sample_cpu_last_proc = 0;
-static double _sample_cpu_usage = 0.0;
-static bool _sample_cpu_enabled = true;
```

Comment:
Use a separate `proc` utils to be able to use is on linux and android, but android also needs a separate implementation for Android 8+